assembly/fabric8-ide-dashboard-war/pom.xml

```diff
@@ -226,9 +254,15 @@
                                 <configuration>
                                     <target>
                                         <!-- build user dashboard -->
-                                        <exec dir="${basedir}/target/sources" executable="node_modules/yarn/bin/yarn" failonerror="true">
-                                            <arg value="install" />
+                                        <exec dir="${basedir}/target/sources" executable="yarn" failonerror="true">
+                                            <arg value="build" />
                                         </exec>
+                                        <!-- Change base HREF of the application that will be hosted on /dashboard
```

Comment:
Can we remove commented section? I think it is already enabled above

Reply:
Done